raycicmd/converter_test.go

```diff
@@ -260,6 +261,36 @@ func TestConvertPipelineStep(t *testing.T) {
 				"BUILDKITE_ARTIFACT_UPLOAD_DESTINATION": artifactDest,
 			},
 		},
+	}, {
+		in: map[string]any{
+			"label":         "mac job",
+			"key":           "mac",
+			"command":       "echo mac",
+			"job_env":       "MACOS",
+			"instance_type": "macos",
+		},
+		out: map[string]any{
+			"label":   "mac job",
+			"key":     "mac",
+			"command": "echo mac",
+			"agents":  newBkAgents("fakemacrunner"),
+
+			"timeout_in_minutes": defaultTimeoutInMinutes,
+			"retry":              defaultRayRetry,
+			"env": map[string]string{
+				"BUILDKITE_ARTIFACT_UPLOAD_DESTINATION": "s3://artifacts_bucket/abcdefg1234567890",
+				"BUILDKITE_BAZEL_CACHE_URL":             "https://bazel-build-cache",
+				"RAYCI_BRANCH":                          "beta",
+				"RAYCI_BUILD_ID":                        "abc123",
+				"RAYCI_TEMP":                            "s3://ci-temp/abc123/",
+				"RAYCI_WORK_REPO":                       "fakeecr",
+			},
+			"plugins": []any{map[string]any{
+				macosSandboxPlugin: map[string]string{
+					"deny-file-read": "/usr/local/etc/buildkite-agent/buildkite-agent.cfg",
+				},
+			}},
```

Comment:
please remove this and check plugin from the extracted `plugins`. to be consistent with the semantics of other `out` values.